### ISSUE TITLE
Labels cause e2e tests to fail via accessibility errors [9]

### ISSUE THREAD

**Issue by gurjmatharu** [bcgov/cas-cif#933]
Describe the Bug:

Probability (how likely the bug is to happen, scored from 1-5): 4
(For example, probability of 5 is something like "it happens to all users every time they log in."
Probability of 1 "only happens to certain users when a really specific and unlikely path is followed.")

Effect (how bad the bug is when it does happen, scored from 1-5): 5
(For example, effect of 5 is "the entire app crashes and makes it unusable for all users" or "the bug causes the wrong data to be saved, with critical information (e.g. payment) being affected."
Effect of 1 is "It makes some styling look a little bit weird.")

Steps to reproduce the behaviour: 
There is an assertion and wait statement near https://github.com/bcgov/cas-cif/blob/develop/app/cypress/integration/cif/project-revision/edit-project-revision.spec.js#L121 .  Without these the e2e throws accessibility errors.  These errors are caused by the rendering of the label of on track, etc.

Screenshots:

Additional information:

- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]
- Additional context

**Comment by Sepehr-Sobhani** [comment]
Hey @gurjmatharu! I've removed the `wait` command and it's not causing an issue on my end; Can you try that on your end as well to see if this issue still exists?!

**Comment by Sepehr-Sobhani** [comment]
@LindsayMacfarlane Moved this bug into `Accepted by PO or Dev` as it contains only code changes and nothing to test.
